_emails/newsletter-2021-03.html

```diff
@@ -0,0 +1,148 @@
+---
+title: Origami Newsletter, March 2021
+companion_post_url: https://origami.ft.com/blog/2021/04/12/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+Updated Origami workshops. Work has begun to migrate Origami to npm. Reminder, we deprecate FT's Bower Registry in July. 2021 and plan to decomission in July 2022.
+{% endcapture %}
+
+{% include email/tldr.html content=text %}
+
+<!-- Top things -->
+{% include email/h2.html content="Top Things" %}
+
+{% capture text %}Some of the bigger Origami news from the last month:{% endcapture %}
+{% include email/markdown.html content=text %}
+
+
+
+<!-- Thing 1 -->
+{% include email/h3.html content="Update on the migration to npm (dropping Bower support)" %}
+{% capture text %}
+During January we announced our plan to [decommission the FT Bower Registry and migrate to the public npm registry](https://origami.ft.com/blog/2021/01/18/deprecating-bower-and-origami-via-npm/#who-does-this-affect).
+
+Work has now started on migrating the Origami components onto npm, 8 out of 66 of our component are now ready to be migrated. We aim to have all components, including documentation/guidance for how to migrate your products, complete by the start of Q3.
```

Comment:
maybe "are now migrated, ready to be released" ?

Reply:
ooh yes, this is better and more accurate ☺️